x-pack/metricbeat/module/aws/ec2/_meta/data.json

```diff
@@ -0,0 +1,64 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "aws": {
+        "ec2": {
+            "cpu": {
+                "total": {
+                    "pct": 1.9520833333372138
+                }
+            },
+            "diskio": {
+                "read": {
+                    "bytes": 1313177.6,
+                    "ops": 241.4
+                },
+                "write": {
+                    "bytes": 35465625.6,
+                    "ops": 1809
+                }
+            },
+            "network": {
+                "in": {
+                    "bytes": 42,
+                    "packets": 1
+                },
+                "out": {
+                    "bytes": 28,
+                    "packets": 1
+                }
+            },
+            "status": {
+                "check_failed": 0,
+                "check_failed_instance": 0,
+                "check_failed_system": 0
+            }
+        }
+    },
+    "cloud": {
```

Comment:
This should be mapped to ECS: https://github.com/elastic/ecs#cloud

Reply:
@ruflin I think I followed the ecs naming here already hmmm Sorry can you give me an example which part didn't map right please :-)

Reply:
Sorry, my bad here. I thought it is under `aws` but it is on the top level. All good, sorry for the noise.